packages/shared/src/components/modals/award/GiveAwardModal.tsx

```diff
@@ -216,11 +216,7 @@ const CommentScreen = () => {
   const [note, setNote] = useState('');
 
   const { mutate: awardMutation, isPending } = useMutation({
-    mutationKey: [
-      'awards',
-      { productId: product?.id, type, entityId: entity.id, note },
-    ],
-    mutationFn: award,
+    ...awardMutationOptions(),
```

Comment:
Same as query mutation function we also do the same for mutations. We can then reuse the function in `useQuerySubscription` to listen to mutation events from other places avoiding props drilling and options passing all over the place.